src/config/media/pipewire.md

```diff
@@ -0,0 +1,87 @@
+# PipeWire
+
+To use PipeWire, install the `pipewire` package.
+
+PipeWire should be started as a user. Use the autostarting mechanism of your
+desktop environment.
+
+If using [startx](../graphical-session/xorg.md#startx), add the following line
+in your `.xinitrc` file:
+
+```
+pipewire &
+```
+
+If using [Sway](https://swaywm.org/), start it from the configuration file:
+
+```
+exec pipewire
+```
+
+## PulseAudio replacement
+
+PipeWire can start a PulseAudio server. While it's possible to modify
+`/etc/pipewire/pipewire.conf` for auto-starting the PulseAudio server, it's
+recommended to start it from the same place where you start PipeWire by
+executing `pipewire-pulse`. This keeps PipeWire's configuration file unmodified
+for smoother future upgrades.
+
+#### Example for `.xinitrc`
+
+```
+pipewire &
+pipewire-pulse &
+```
+
+#### Example for Sway
+
+```
+exec pipewire
+exec pipewire-pulse
+```
+
+After setting up `pipewire-pulse`, make sure that the PulseAudio service is
+[disabled](../services/index.md#disabling-services) and reboot.
+
+To check if the replacement is working correctly, use the following command:
+
+```
+$ pactl info
+
+...
+Server Name: PulseAudio (on PipeWire 0.3.18)
+...
+```
+
+##### Note on Bluetooth audio
+
+If having troubles with Bluetooth audio, make sure that `libspa-bluetooth`
+package is installed.
```

Comment:
The package is required for bluetooth at all, so I don't think this is the correct language.

And title should be just `Bluetooth audio`

Reply:
It was required for bluetooth audio to work in my case. But let me know if we should remove this.

Reply:
What @ericonr is saying is that "If having troubles" is not the correct language when libspa-bluetooth is required for bluetooth audio to work.

Reply:
Oh I misread that, I thought @ericonr said "is _not_ required for bluetooth at all" :)

Reply:
updated